samtranslator/internal/deprecation_control.py

```diff
@@ -0,0 +1,67 @@
+"""
+Utils for deprecating our code using warning.warn().
+The warning message is written to stderr when shown.
+
+For the difference between DeprecationWarning
+and PendingDeprecationWarning, refer to
+https://peps.python.org/pep-0565/#additional-use-case-for-futurewarning
+
+If external packages import deprecated/pending-deprecation
+interfaces, it is their responsibility to detect and remove them.
+"""
+import warnings
+from functools import wraps
+from typing import Callable, Optional, TypeVar
+
+RT = TypeVar("RT")  # return type
+
+
+def _make_message(message: str, replacement: Optional[str]) -> str:
+    if replacement:
+        return f"{message}, please consider to use {replacement}"
+    return f"{message} and there is no replacement."
```

Comment:
More stern, to the point:

```suggestion
        return f"{message}, please use {replacement}"
    return message
```

Reply:
updated